stubs/ldap3/ldap3/utils/asn1.pyi

```diff
@@ -1,37 +1,49 @@
-from typing import Any
-from typing_extensions import TypeAlias
+from _typeshed import Incomplete, IndexableBuffer, SliceableBuffer
+from collections.abc import Callable, Mapping
+from typing import Any, TypeVar, overload
+from typing_extensions import Final, TypeAlias
 
-# Enable when pyasn1 gets stubs:
-# from pyasn1.codec.ber.encoder import AbstractItemEncoder
-_AbstractItemEncoder: TypeAlias = Any
+from pyasn1.codec.ber.encoder import AbstractItemEncoder, tagMap, typeMap
 
-CLASSES: Any
+# Use _typeshed.SupportsGetItemBuffer after PEP 688
+SupportsGetItemBuffer: TypeAlias = SliceableBuffer | IndexableBuffer
+_Unused: TypeAlias = object
```

Comment:
Proposal to make this a helper TypeAlias in _typeshed? (plus it's time to do it before mypy updates its vendored _typeshed with a new release)

Reply:
Sure :)